org.eclipse.languageserver/src/org/eclipse/languageserver/languages/InitializeLaunchConfigurations.java

```diff
@@ -130,7 +130,7 @@ public void earlyStartup() {
 				//     (don't forget the `npm run watch` after the install script)
```

Comment:
adding --stdio makes other comments on line 94 and 127 irrelevant. Can you please either fix them, or if there is nothing necessary to comment, remove them.